src/components/groupList/index.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import React from "react";
+import { useState, useEffect } from "react";
+import { Grupo } from "@/types/Grupo";
+import styles from "./groupList.module.css";
+import FormTitle from "@/components/title/formTitle";
+import { useGroupProvider } from "@/contexts/groups/groups";
+import ModalGroup from "@/components/modalGrupo";
+import FormContainer from "@/components/ui/formContainer";
+import MuiButton from "@mui/material/Button";
+
+const GrupoList = () => {
+  const [openGroupModal, setOpenGroupModal] = useState<boolean>(false);
+  const [currentGroup, setCurrentGroup] = useState<Grupo | null>(null);
+  const [filtroStatus, setFiltroStatus] = useState<'todos' | 'ativos' | 'inativos'>('todos');
+  
+  const {
+    groupList,
+    getGroups
+  } = useGroupProvider();
+
+  useEffect(() => {
+    getGroups();
+  }, []);
+
+  const handleGroupCreated = () => {
+    getGroups();
+  };
+  
+  const getGruposFiltrados = () => {
+    if (!Array.isArray(groupList)) return [];
+    
+    switch (filtroStatus) {
+      case 'ativos':
+        return groupList.filter(grupo => grupo.isAtivo);
+      case 'inativos':
+        return groupList.filter(grupo => !grupo.isAtivo);
+      default:
+        return groupList;
+    }
+  };
+
+  const gruposFiltrados = getGruposFiltrados();
+
+  return (
+    <FormContainer>
+      <div className="flex flex-col justify-items-center w-full">
+        <div className="w-full mb-6 flex flex-col md:flex-row md:items-center">
+          <div className="w-full flex justify-center mb-4 md:mb-0 md:absolute md:left-0 md:right-0">
+            <FormTitle
+              title="Grupos"
+              className={`rounded p-2 z-10`}
```

Comment:
```suggestion
              className="rounded p-2 z-10"
```

Usar `` apenas para concatenar variáveis.